core/src/main/java/org/mskcc/cbio/portal/scripts/ImportTreatments.java

```diff
@@ -109,15 +109,10 @@ public void run() {
     *
     * @param updateInfo
     */
-    public static void startImport(OptionSet options, OptionSpec<String> data, boolean updateInfo){
-        try {
-            if (options.hasArgument(data)) {
-                File treatmentFile = new File(options.valueOf(data));
-                importData(treatmentFile, updateInfo);
-            }
-        }
-        catch (Exception ex) {
-            ex.printStackTrace();
```

Comment:
yes, please add the TODO to remove this one in ImportGenesetData

Reply:
Done.